docs/readmes/orc8r/dev_build.md

```diff
@@ -64,11 +64,18 @@ git checkout -b v1.6 origin/v1.6  # or desired Git target, e.g. master
 ```
 
 Build and publish Orchestrator images
+(Optionally, build and publish fluentd images for the Orchestrator by uncommenting the line `docker-compose.logging.yaml` in `build.py`.)
```

Comment:
This seems to be duplicate to line 75 or a comment which refines line 75. Thus it should probably go to line 76, to have all comments on this additional step in one place. It is somewhat confusing otherwise, if we talk in two places about optional steps, when they are essentially one.

Reply:
The reason for splitting it like that was so that `./build.py --all` only needs to be run once, because it takes quite long to complete. But I can see how this might be confusing to read, so I just moved everything about fluentd into one block below.